Formula/vim.rb

```diff
@@ -116,13 +124,24 @@ def install
   end
 
   test do
-    if build.with? "python"
+    if build.with?("python") && build.without?("python3")
```

Comment:
Since you can build `--with-python` and `--with-python3` I think we should do this `if build.with? "python"` regardless of whether you built with or without python3. So let's leave it as
```
    if build.with? "python"
```